src/resistance.c

```diff
@@ -61,6 +61,9 @@ resistance_move_apply(struct view *view, double *x, double *y)
 	}
 
 	wl_list_for_each(output, &server->outputs, link) {
+		if (!output->wlr_output->enabled)
```

Comment:
Please always use curly braces like `if (..) {`, even if there is only a single statement.